caffe2/python/operator_test/momentum_sgd_test.py

```diff
@@ -174,11 +174,13 @@ def momentum_sgd(grad, param_momentum, lr, param=None):
             weight_decay=0.0,
         )
 
+        threshold = 1e-3 if (gc.device_type == caffe2_pb2.HIP) else 1e-4
```

Comment:
@bddppq Is it a valid threshold? pls advise.

Reply:
The (random) inputs could be different on different machines. When the tests failed, "hypothesis" library should print the "seed" for reproduce the same failure, could you paste that here?

Reply:
@petrex I'm hoping you can tell me whether the threshold is valid :-)